client/chain/chain.go

```diff
@@ -656,6 +660,19 @@ func (c *chainClient) runBatchBroadcast() {
 	}
 }
 
+func (c *chainClient) GetGasFee() (string, error) {
+	gasPrices := strings.Trim(c.opts.GasPrices, "inj")
+
+	gas, _ := strconv.ParseFloat(gasPrices, 18)
```

Comment:
1. ParseFloat(string, bitSize). bitSize should be 32 or 64, shouldn't be 18
2. We should handle error, ie:
```
gas, err  := strconv.ParseFloat(gasPrices, 64)
if err != nil { 
   return "", err
}
```

Basically if we fetch from chain, there won't be error here, but should handle as it is safer for users